src/app/Shared/Selector.component.tsx

```diff
@@ -94,7 +94,7 @@ export interface SelectorProps {
     /**
      * values that are already selected in other selectors with labels
      */
-    selectedValuesObj: { value: string; label: string }[];
+    selectedValuesObj: { value: string; label: string }[] | null;
```

Comment:
I suggest to keep this input prop clean. So empty array is sufficient to indicate that nothing is selected and null is kind of obsolete.

Reply:
Done.